playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -24,13 +25,29 @@
       instances: "{{ master_names }}"
       cluster: "{{ cluster_id }}"
       type: "{{ k8s_type }}"
+      g_sub_host_type: "default"
 
   - include: ../../common/openshift-cluster/set_node_launch_facts_tasks.yml
+    vars:
+      type: "node"
+      count: "{{ num_nodes }}"
+  - include: tasks/launch_instances.yml
+    vars:
+      instances: "{{ node_names }}"
+      cluster: "{{ cluster_id }}"
+      type: "{{ k8s_type }}"
+      g_sub_host_type: "{{ sub_host_type }}"
```

Comment:
Is this working correctly, I don't see sub_host_type specified before this is used, and it looks like the launch_instances playbook is setting sub_host_type to g_sub_host_type.